worlds/husarion_world.sdf

```diff
@@ -7,36 +7,80 @@
       <real_time_factor>1.0</real_time_factor>
     </physics>
 
-    <light type="directional" name="sun">
-      <cast_shadows>true</cast_shadows>
-      <pose>0 0 10 0 0 0</pose>
-      <diffuse>0.8 0.8 0.8 1</diffuse>
-      <specular>0.2 0.2 0.2 1</specular>
-      <attenuation>
-        <range>1000</range>
-        <constant>0.9</constant>
-        <linear>0.01</linear>
-        <quadratic>0.001</quadratic>
-      </attenuation>
-      <direction>-0.5 0.1 -0.9</direction>
-    </light>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="gz::sim::systems::Physics">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-user-commands-system"
+      name="gz::sim::systems::UserCommands">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-scene-broadcaster-system"
+      name="gz::sim::systems::SceneBroadcaster">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-contact-system"
+      name="gz::sim::systems::Contact">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-imu-system"
+      name="gz::sim::systems::Imu">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-sensors-system"
+      name="gz::sim::systems::Sensors">
+    </plugin>
 
-    <model name="ground_plane">
+    <model name="husarion_world">
       <static>true</static>
-      <link name="link">
+      <link name="mesh">
+        <pose>13.0 16.0 0 0 0 -1.57</pose>
+        <visual name="visual">
+          <geometry>
+            <mesh>
+              <uri>model://HusarionLogo/meshes/husarion_logo.dae</uri>
+              <scale>6.0 6.0 6.0</scale>
+            </mesh>
+          </geometry>
+        </visual>
+        <collision name="visual">
+          <geometry>
+            <mesh>
+              <uri>model://HusarionLogo/meshes/husarion_logo.dae</uri>
+              <scale>6.0 6.0 6.0</scale>
+            </mesh>
+          </geometry>
+        </collision>
+      </link>
+
+      <link name="map">
+        <light type="directional" name="sun">
+          <cast_shadows>true</cast_shadows>
+          <pose>0 0 10 0 0 0</pose>
+          <diffuse>0.8 0.8 0.8 1</diffuse>
+          <specular>0.2 0.2 0.2 1</specular>
+          <attenuation>
+            <range>1000</range>
+            <constant>0.9</constant>
+            <linear>0.01</linear>
+            <quadratic>0.001</quadratic>
+          </attenuation>
+          <direction>-0.5 0.1 -0.9</direction>
+        </light>
         <collision name="collision">
           <geometry>
             <plane>
               <normal>0 0 1</normal>
-              <size>100 100</size>
+              <size>15 15</size>
             </plane>
           </geometry>
         </collision>
         <visual name="visual">
           <geometry>
             <plane>
               <normal>0 0 1</normal>
-              <size>100 100</size>
+              <size>50 50</size>
```

Comment:
You forgot about visual. Also isn't 15 too small? The grid is 20x20 so the map I guess will be around 25